manifests/params.pp

```diff
@@ -10,6 +10,7 @@
       $configdir     = '/etc'
       $root_group    = 'root'
       $logrotate_bin = '/usr/sbin/logrotate'
+      $logrotate_state = '/var/lib/logrotate/logrotate.status'
```

Comment:
The default should probably be `undef`, (to preserve existing behaviour) and be outside of the `case` block?